Customer reported Huawei E3372 not working
anymore starting with balenaOS v2.32.
Fix for this appears to be switching Huawei E3372
modem with vendor and product numbers 12d1:1f01
from the cdc_ether legacy mode to the faster mbim mode.

Upstream-status: Inappropriate [configuration]
Signed-off-by: Alexandru Costache <alexandru@balena.io>

Index: usb-modeswitch-data-20170806/usb_modeswitch.d/12d1:1f01
===================================================================
--- usb-modeswitch-data-20170806.orig/usb_modeswitch.d/12d1:1f01
+++ usb-modeswitch-data-20170806/usb_modeswitch.d/12d1:1f01
@@ -1,6 +1,6 @@
 # Huawei E353 (3.se) and others
 TargetVendor=0x12d1
 TargetProductList="14db,14dc"
-HuaweiNewMode=1
-
+#HuaweiNewMode=1
+MessageContent="55534243123456780000000000000011063000000100010000000000000000"
